examples/bpk-component-graphic-promotion/stories.js

```diff
@@ -30,11 +31,12 @@ import {
 } from './examples';
 
 storiesOf('bpk-component-graphic-promotion', module)
-  .add('Default styling', NoStyleExample)
-  .add('Default', DefaultExample)
+  .add('Default no background image', NoStyleExample)
```

Comment:
Do we ever expect the GP to be used without a Background image or is it always expected?

Reply:
Well out of the box it won't have a background image, I don't know if that would ever be valuable but I think we should be aware what the out-of-the-box component looks like.